Utils/ProcessArtifactUtils.cpp

```diff
@@ -0,0 +1,37 @@
+#include "pch.h"
+#include "ProcessArtifactUtils.h"
+
+std::wstring ProcessArtifactUtils::getFakeFilePath(std::string fileName)
+{
+    wchar_t tempPath[MAX_PATH + 1];
```

Comment:
Use std::vector instead, you can find example here:
https://github.com/AvivShabtay/Stresser/blob/d7bb59d3af088bfc3591f99105ac282933e8314e/Utils/LocalPcUtils.cpp#L8

SUGGESTION: Maybe move this function to `Utils` as helper function, under `LocalPcUtils`